src/main/java/com/autotune/experimentManager/core/ExperimentTrialHandler.java

```diff
@@ -177,6 +181,8 @@ public void startExperimentTrials() {
                         switch (loadAvailabilityStatus) {
                             case LOAD_AVAILABLE:
                                 // Proceed to collect metrics as load is available
+                                // Setting the status to `COLLECTING_METRICS` as the load is available
+                                this.experimentTrial.setStatus(EMUtil.EMExpStatus.COLLECTING_METRICS);
```

Comment:
Same is the case here, cannot assume anything about METRICS. We can only state that LOAD is available or not and accordingly set the status